File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main (void)
{
    int cash;
    //get cash
    do{
        cash = round(get_float("Change owed: ")*100);
    }
    while (cash<0);
    //define coins as variables
    int quarters = 25;
    int dimes = 10;
    int nickels = 5;
    int pennies = 1;
    //initialize varible q of each coin
    int n_qrt = 0;
    int n_dim = 0;
    int n_nik = 0;
    int n_pen = 0;
    int mod_qrt = (int) (cash % quarters);
    n_qrt = cash / quarters;
    n_dim = mod_qrt / dimes;
    int mod_dim = (int) ( (int) (cash % quarters) % dimes);
    n_nik = mod_dim / nickels;
    int mod_nik = (int) round((int) round((int) round(cash % quarters) % dimes) % nickels);
    n_pen = mod_nik / pennies;
    int coins = n_qrt + n_dim + n_nik + n_pen;
    printf("%i\n",coins);
}

File: pset6/cash/cash.py
# Import library
from cs50 import get_int, get_float

# Get cash
cash = get_float("Change owed: ")

# Check cash greater than 0
while True:
    if cash > 0:
        break
    else:
        # Else prompt again
        cash = get_float("Change owed: ")

# Define coins
quarters = 0.25
dimes = 0.10
nickels = 0.05
pennies = 0.01

# Define number of each coins to return
n_qrt = 0
n_dim = 0
n_nik = 0
n_pen = 0

# Calculate number of coins
# Quarters
n_qrt = cash // quarters
mod_qrt = round(cash % quarters, 2)

# Dimes
n_dim = mod_qrt // dimes
mod_dim = round(round(cash % quarters, 2) % dimes, 2)

# Nickels
n_nik = mod_dim // nickels
mod_nik = round(round(round(cash % quarters, 2) % dimes, 2) % nickels, 2)

# Pennies
n_pen = mod_nik // pennies
coins = int(n_qrt + n_dim + n_nik + n_pen)

# Print number of coins to give back
print(coins)